addons/advanced_fatigue/functions/fnc_handleStaminaBar.sqf

```diff
@@ -13,6 +13,11 @@ params ["_stamina"];
 
 private _staminaBarContainer = uiNamespace getVariable [QGVAR(staminaBarContainer), controlNull];
 
+if ((vehicle ACE_player) != ACE_player) exitWith { // Hide when mounted
```

Comment:
You could do it just like in the visibleMap EH. Both need checks if both conditions apply. I'd rather keep all the hiding/showing in one place.